test/commonjs/autohooks/routes/.autohooks.js

```diff
@@ -1,7 +1,6 @@
-module.exports = async function (app, opts, next) {
```

Comment:
There were tons of warning indicating deprecation for async/callback mix but CI currently don't fail.